CHANGELOG.md

```diff
@@ -37,15 +40,51 @@ Changes by Version
     * `--cassandra.enable-dependencies-v2` - Jaeger will automatically detect the version of the dependencies table
     * `--cassandra.tls.verify-host` - please use `--cassandra.tls.skip-host-verify` instead
 
+* Remove incorrectly scoped downsample flags from the query service ([#2782](https://github.com/jaegertracing/jaeger/pull/2782), [@joe-elliott](https://github.com/joe-elliott))
+    * `--downsampling.hashsalt` removed from jaeger-query
+    * `--downsampling.ratio` removed from jaeger-query
+    
 #### New Features
 
-* Add TLS Support for gRPC and HTTP endpoints of the Query server ([#2337](https://github.com/jaegertracing/jaeger/pull/2337), [#2772](https://github.com/jaegertracing/jaeger/pull/2772), [@rjs211](https://github.com/rjs211))
+* Add TLS Support for gRPC and HTTP endpoints of the Query and Collector servers ([#2337](https://github.com/jaegertracing/jaeger/pull/2337), [#2772](https://github.com/jaegertracing/jaeger/pull/2772), [#2798](https://github.com/jaegertracing/jaeger/pull/2798), [@rjs211](https://github.com/rjs211))
```

Comment:
I've merged #2337, #2772 and #2798 together in this bullet given they're closes related and only differ by the backend component.